src/libs/brand-utils.test.js

```diff
@@ -50,4 +50,38 @@ describe('isBrand', () => {
     // Assert
     expect(isTestBrand).toBe(true)
   })
+
+  it('getEnabledBrand returns forced brand when a valid one is set', () => {
+    // Arrange
+    window.configOverridesStore.set({ brand: 'rareX' })
+
+    // Act
+    const enabledBrand = getEnabledBrand()
+
+    // Assert
+    expect(enabledBrand).toEqual(brands.rareX)
```

Comment:
Suggest using [`toBe`](https://jestjs.io/docs/expect#tobevalue) instead of `toEqual` here since the `isAnvil`, `isBaseline`, etc. functions rely on this function returning the brand object instead of a clone.

Reply:
Done in [8125a1d](https://github.com/DataBiosphere/terra-ui/pull/3625/commits/8125a1d0a54da2b99060b12cca2673bed1da9b79).